content/install/installation_reference.md

```diff
@@ -2,10 +2,132 @@
 date = "2017-04-15T14:39:04+02:00"
 title = "Installation Reference"
 url = "installation-reference"
-draft = true
 
 [menu.install]
   weight = 2
   identifier = "installation-reference"
   parent = "install_overview"
 +++
+
+{{% alert warn %}}
+Some of these options may be experimental.
+
+Options are auto-generated from the source code: https://github.com/drone/drone/tree/master/cmd
+{{% /alert %}}
+
+# Drone Server
+
+This is the reference list of all environment variables available to configure Drone Server.
+
+CLI FLAG <br><br> ENV VAR | DESCRIPTION <br><br> DEFAULT
+--- | ---
+`--debug--debug` <br><br> `$DRONE_DEBUG` | start the server in debug mode <br><br> ` `
```

Comment:
Seems some issue in parsing causing flags that do not take a value to show double.  There are other options below doing the same.

Reply:
Good catch, I will revist the regex tomorrow.